airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostresSourceComprehensiveTest.java

```diff
@@ -346,6 +346,14 @@ protected void initTests() {
             .addExpectedValues("99999", "5.1", "0", null)
             .build());
 
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .fullSourceDataType("numeric(13,4)")
+            .airbyteType(JsonSchemaPrimitive.NUMBER)
+            .addInsertValues("0.1880", "null")
+            .addExpectedValues("0.1880", null)
+            .build());
+
```

Comment:
This file does the datatype tests for non-CDC syncs so no need to add in here :)